pandoc/pandoc-zotero-live-citemarkers.lua

```diff
@@ -221,6 +284,11 @@ function Meta(meta)
     config.csl_style = pandoc.utils.stringify(meta.zotero['csl-style'])
   end
 
+  config.transferable = test_boolean('transferable', meta.zotero['transferable'])
+
+  -- refuse to create a transferable document, when csl style is not specified
+  config.transferable = config.transferable and config.csl_style
```

Comment:
I'd prefer a call to `error(...)` here rather than silently ignoring it.

Reply:
> I'd prefer a call to `error(...)` here rather than silently ignoring it.

It will easier if you do it.   (I am not familiar with Lua much).  Ignoring it doesn't break anything, you can merge the change, mark it as WIP, and come back to it later.

Like scannable cite documents, for documents without styles, the 'Export to other word processors ...' doesn't apply.

![Screenshot from 2020-07-30 18-26-07](https://user-images.githubusercontent.com/19869864/88925622-60d74400-d292-11ea-91b1-cce28b2f5e7d.png)

Reply:
Got it.